src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerskort.tsx

```diff
@@ -0,0 +1,62 @@
+import './fagsakdeltagerskort.less';
+
+import React from 'react';
+
+import { Infokort } from '@navikt/familie-header';
+import {
+    GuttIkon, JenteIkon, KvinneIkon, MannIkon, NøytralPersonIkon
+} from '@navikt/familie-ikoner';
+import { kjønnType } from '@navikt/familie-typer';
+
+import { FagsakDeltagerRolle } from '../../../typer/fagsakdeltager';
+
+export interface IFagsakDeltagerskortProps {
+    navn: string;
+    ident: string;
+    rolle: FagsakDeltagerRolle;
+    index: number;
+    kjønn?: kjønnType;
+    fagsakId?: string;
+    onClick?: (index: number) => void;
+    children?: React.ReactNode | React.ReactNode[];
+}
+
+const FagsakDeltagerskort: React.FunctionComponent<IFagsakDeltagerskortProps> = ({
+    navn, ident, rolle, index, kjønn, fagsakId, onClick, children
+}) => {
+    type IkonerMap = Record<string, React.ReactNode>;
+    type RolleNavnMap = Record<string, string>;
+
+    const ikoner: IkonerMap = {
+        "FORELDER_MANN": <MannIkon className='ikon' />,
+        "FORELDER_KVINNE": <KvinneIkon className='ikon' />,
+        "FORELDER_UKJENT": <NøytralPersonIkon className='ikon' />,
+        "BARN_KVINNE": <JenteIkon className='ikon' />,
+        "BARN_MANN": <GuttIkon className='ikon' />,
+        "BARN_UKJENT": <NøytralPersonIkon className='ikon' />
+    };
+
+    const rolleNavn: RolleNavnMap = {
+        "FORELDER_MANN": "FAR",
+        "FORELDER_KVINNE": "MOR",
+        "FORELDER_UKJENT": "FORELDER",
+        "BARN_KVINNE": "BARN",
+        "BARN_MANN": "BARN",
+        "BARN_UKJENT": "BARN"
+    }
+
+    return (
+        <div className={`${fagsakId ? "" : "gray-out"}`}>
```

Comment:
What does this do?

Reply:
According to design, if a person has no case, his/her card should be displayed as gray out

Reply:
Ok, maybe add a variable or something so that it is easier to understand.